Automattic/wp-calypso
# Editing Toolkit Sidebar: Update design

## Changes proposed in this Pull Request

Updating the sidebar to match the design in https://github.com/Automattic/wp-calypso/issues/45240

![Aug-28-2020 15-28-06](https://user-images.githubusercontent.com/6458278/91524686-27eac780-e943-11ea-95ab-55eea89b5f5c.gif)

I left the draft/scheduled badges in there to distinguish between the two.

## Testing instructions

Tested in Chrome, Safari and IE Edge

Apply D48670-code to your sandbox

Sandbox a site and edit a page/post

Check that there is no regression in carried-over functionality 

Check that the design matches the updated screenshot in https://github.com/Automattic/wp-calypso/issues/45240

Tested with a long site title, e.g., 
<img width="536" alt="Screen Shot 2020-08-31 at 3 05 22 pm" src="https://user-images.githubusercontent.com/6458278/91684656-6a0e4600-eb9b-11ea-86a0-1d628d71ba66.png">


Fixes #45240
